transport_http.go

```diff
@@ -56,23 +56,27 @@ func (self *transportHTTP) Send() {
 	// unfortunately in case of flush it's unneeded operations but whatever
 	stacks, logs := cloneResetBuffers()
 	if length := len(stacks); length > 0 {
-		stacksRequest := gorequest.New().Post(fmt.Sprintf("%s/%s", STKLOG_HOST, STKLOG_STACKS_ENDPOINT)).Set("Stklog-Project-Key", self.GetProjectKey()).
-			Set("Content-Type", "application/json")
-		stacksRequest.Transport.DisableKeepAlives = true
+
 		for i := 0; i < length; i += min(self.batchSize, length-i) {
+			stacksRequest := self.prepare()
 			execRequest(stacksRequest, stacks[i:min(self.batchSize+i, length)])
 		}
 	}
 	if length := len(logs); length > 0 {
-		logsRequest := gorequest.New().Post(fmt.Sprintf("%s/%s", STKLOG_HOST, STKLOG_LOGS_ENDPOINT)).Set("Stklog-Project-Key", self.GetProjectKey()).
-			Set("Content-Type", "application/json")
-		logsRequest.Transport.DisableKeepAlives = true
+		logsRequest := self.prepare()
 		for i := 0; i < length; i += min(self.batchSize, length-i) {
 			execRequest(logsRequest, logs[i:min(self.batchSize+i, length)])
 		}
 	}
 }
 
+func (self *transportHTTP) prepare() *gorequest.SuperAgent {
+	objectRequest := gorequest.New().Post(fmt.Sprintf("%s/%s", STKLOG_HOST, STKLOG_STACKS_ENDPOINT)).Set("X-Stklog-Project-Key", self.GetProjectKey()).
```

Comment:
You need to pass a parameter to this method, since we want to POST logs or stacks, we have two constants for the endpoints : STKLOG_STACKS_ENDPOINT and STKLOG_LOGS_ENDPOINT.
in your prepare, right now it's using only STACKS